infrastructure/httpapi/handlers/bridges.go

```diff
@@ -154,11 +179,34 @@ func (handler *Bridges) ListActivitiesByNetwork(ctx *fasthttp.RequestCtx) {
 	}
 
 	filter := bridge_activitiy_view.BridgeActivitiesListFilter{
+		MaybeIdGt:        nil,
+		MaybeCreatedAtGt: nil,
+		MaybeUpdatedAtGt: nil,
+	}
+	if queryArgs.Has("id.gt") {
```

Comment:
in this way, `id.gt`, `createdAt.gt` and `updatedAt.gt` will be maturely exclusion. Do we intended to do?

Reply:
Yes this is expected behaviour. It is intended for downstream to scan through the records. But I think there's no harm for me to have them work in parallel. Let me update it.